README.md

```diff
@@ -112,13 +113,21 @@ The very same dialog may also be opened with a link that's present in the notifi
 #### Delete Resources
 You may delete any resource that is listed in the resource tree by choosing "Delete" in the context menu.
 
-#### Follow logs
+#### Follow Logs
 
 IntelliJ-Kubernetes extension allows you to follow logs of the cluster resources such as Pods, Jobs and Builds.
 There is a dedicated tab to display the log messages for the selected Kubernetes resource.
 
 ![logs](images/logs.png)
 
+#### Open Terminals
+
+You can open a terminal to a container that's running on the connected cluster.
+You select a resource such as a Pod, a Job or a Build and choose "Terminal" in the context menu.
```

Comment:
The user can open the Kubernetes resources such as Pod, Build and open that in the Terminal Shell using the context menu action `Terminal`.